CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @superset-ui : Change Log
 
+## v0.8.0
```

Comment:
There are also other commits since `0.7.2` that should be added here.

Reply:
Good point, I've added those now. LMK if I misinterpreted the commit comments or misclassified anything.